### PR TITLE
Add possibility to add links to entries and add some links

### LINKED ISSUE #176: [Feature] Linking to IRIS room usage data
**Is your feature request related to a problem? Please describe.**
Our data is currently quite static, but we could at least link to real time information if there are websites that provide them.

**Describe the solution you'd like**
A link in the info card to https://www.devapp.it.tum.de/iris/app/ for the relevant buildings (and maybe rooms).

**Describe alternatives you've considered**
We could try to directly fetch the data from IRIS into NavigaTUM. But this could be a feature in the future, linking is easier to implement and could already be helpful.

**Additional context**
There are also other helpful sites to link to:
- https://www.ub.tum.de/arbeitsplaetze-auslastung
- https://tum-dev.github.io/eat-api/#!/en/mensa-garching
- https://mensa.liste.party/

### PR DESCRIPTION
**Pull request by octycs**
Resolves #176 in the simplest way. What is not included is the links to the library occupation, because we actually don't have entries for some of the libraries (They are usually a group of several rooms).

We still don't really have API localization (see #168), but I thought it would make sense to make the source data already multilingual. This way the data is already there.

Screenshot:
image

### CODE REVIEW

File: data/processors/export.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def export_for_api(data, path):
         if "roomfinder_data" in export_data[_id]:
             del export_data[_id]["roomfinder_data"]
         if "props" in export_data[_id]:
-            to_delete = list(filter(lambda e: e != "computed", export_data[_id]["props"].keys()))
+            to_delete = list(filter(lambda e: e not in {"computed", "links"}, export_data[_id]["props"].keys()))
```
Comment (CommanderStorm) [inline data/processors/export.py:90]: I always find these filter expressions really hard to parse. What do you think?
```suggestion
            to_delete = [e for e in export_data[_id]["props"].keys() if e not in {"computed", "links"}]
```

File: openapi.yaml [hunk 2]
```diff
@@ -907,6 +911,33 @@ components:
...
+        url:
+          type: object
+          properties:
+            de:
+              type: string
+            en:
+              type: string
+          required:
+            - text
+            - url
```
Comment (CommanderStorm) [inline openapi.yaml:937]:
```suggestion
            - de
            - en
```

File: openapi.yaml [hunk 3]
```diff
@@ -907,6 +911,33 @@ components:
...
+        text:
+          type: object
+          properties:
+            de:
+              type: string
+            en:
+              type: string
+          required:
+            - text
+            - url
```
Comment (CommanderStorm) [inline openapi.yaml:927]:
```suggestion
            - de
            - en
```

**Review by CommanderStorm — APPROVED** [review]
Overall, this is a good addition. I have made some minor comments, but otherwise this can be merged

From an API Perspective, I would prefer it, if we were to separate the de/en on the server and just serve what is appropriate...
I can work on this, once my exams are done... 😅

**Comment by octycs** [comment]
> From an API Perspective, I would prefer it, if we were to separate the de/en on the server and just serve what is appropriate...

Yes, a general support of multiple languages in the API would be better ... if you want work on this that would be great
